app/renderer/js/pages/preference/general-section.ts

```diff
@@ -418,6 +426,50 @@ export default class GeneralSection extends BaseSection {
 		});
 	}
 
+	setLocale(): void {
+		const langDiv: HTMLSelectElement = document.querySelector('.lang-div');
+		const locales: {[key: string]: string} = {
+			bg: 'Bulgarian',
```

Comment:
@manavmehta I was thinking about how we can sync this up with the translations we have in `/translations`. Maybe it's worth putting this in lang.json or something and then import it. cc @abhigyank

Reply:
Hmm yes, then the languages would be dynamic. But the issue would be that `/translations` only has the `key` in the file name, not sure how we can get the language string from there if we are doing some sort of sync.

Reply:
@akashnimare @abhigyank checkout [1219be8](https://github.com/zulip/zulip-desktop/pull/902/commits/1219be8ac7e4b023c3e8d0dd94d6edfcb4c34c15)
I hope this is what you're looking for @akashnimare ?

Also any refinements?